receiver/vcenterreceiver/metadata.yaml

```diff
@@ -31,6 +31,10 @@ resource_attributes:
     description: The instance UUID of the virtual machine.
     enabled: true
     type: string
+  vcenter.system.device.id:
+    description: The unique identifier of the specific hardware or virtual component being utilized in the vCenter environment.
+    enabled: true
```

Comment:
```suggestion
    enabled: false
```

@djaglowski I know this receiver is still in alpha state, but what's your thoughts on this RA being enabled by default?

I have a slight concern about cardinality increase for current users even if it dropped information previously

Reply:
I'm mostly concerned with whether or not data points are uniquely identifiable. If we disable the attribute, then we can't tell the data points apart, so I think it has to correlate with disabling the data points as well.

That said, would you feel better about it if we switch back to having device id as a data point attribute? I've recently been convinced that this is allowable based on https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23565#issuecomment-1642641080, specifically the suggestion that "We typically put multiple entities in a Resource."

Reply:
I think I'm of the opinion that the first fix for this case might just be to find and only report the datapoint referring to the correct instance of a host or VM. I caught that at least for most of these metrics that there exists a datapoint that correlates correctly to the host system and the VM. So rather than recording on every datapoint, we just find the corresponding correct resource and record only that datapoint for the current metrics.

I think further work beyond that would be to effectively record Network Interfaces, Disks, and other fields of the `Instance` fields on the sampleInfo. 

Like for this example with "net.bytesRx.average",  we can see that there is an already aggregated value for the instance of the VM along with the corresponding Network Interfaces.

<img width="1205" alt="image" src="https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/32067685/11fe28b3-a9f0-4890-9467-718252117991">

So there is more hierarchy here 
| VM
| -- vmnic-0
| -- vmnic-1
| -- vmnic-2
and we have a metric `vcenter.vm.network.throughput`

So that is a peculiarity that is interesting in this case. I believe that the "correct" is to correlate that datapoint only and then we can add more metrics later for the different network interfaces and other instances and create identifying resource attributes on those.

Those are just my opinions though and you probably know a tad bit more about the correctness of the data model so I'll defer to you, but I just wanted to write out my thoughts on what seemed like the correct and least destructive approach. 

Let me know if those ramblings make sense or where we disagree on the approach. I could imagine an alternative approach that we move away from a metric named `vcenter.vm.network.throughput` in favor of `vcenter.vm.network.interface.throughput`, but I'm not sure if that would warrant the removal of `vcenter.vm.network.throughput`.

Reply:
If you're confident that we can isolate only one data point which correctly represents the entity, then let's do that.